contracts/governance/extensions/GovernorTimelock.sol

```diff
@@ -0,0 +1,201 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.19;
+
+import "./IGovernorTimelock.sol";
+import "../Governor.sol";
+import "../../access/manager/IManaged.sol";
+import {IAccessManager} from "../../access/manager/IAccessManager.sol";
+
+/**
+ * @dev TODO
+ *
+ * _Available since v5.0._
+ */
+abstract contract GovernorTimelockCompound is IGovernorTimelock, Governor {
```

Comment:
This needs a name. Tough one. Does it even qualify as a "timelock" type of extension?

Reply:
I would say it does: any call for which there is no enforced delay (by an AccessManager) is subject to the `_defaultDelaySeconds` that dev can customize. Also, operations need to be queued before they can be executed.